Context/Context.php

```diff
@@ -4,9 +4,6 @@
 
 namespace OpenTelemetry\Context;
 
-/**
- * @template TContext of Context
```

Comment:
Going to look into if I can bring this back, but psalm was complaining when trying to pass a context instance to `TraceContextPropagator#inject`.